lib/redcord/actions.rb

```diff
@@ -35,13 +35,18 @@ def create!(args)
 
     sig { params(id: T.untyped).returns(T.untyped) }
     def find(id)
-      instance_key = "#{model_key}:id:#{id}"
-      args = redis.hgetall(instance_key)
-      if args.empty?
-        raise Redcord::RecordNotFound, "Couldn't find #{name} with 'id'=#{id}"
+      Redcord::Base.trace(
```

Comment:
what about tracing create! and find_by and the rest in this file?

Reply:
that's a great idea! we should add tracing to both places. (actions.rb, and prepared_redis.rb (as named redis commands))

Reply:
My intent in suggesting automation (only if feasible and simple!) was to avoid missing Redcord API entrypoints like we might have here.

Reply:
Yeah! Unfortunately seems dynamically inserting span would require some effort 😅  because model_name is not always determined prior to the method definition.